### ISSUE TITLE
SDWebImageDownloaderOptions.lowPriority

### ISSUE THREAD

**Issue by Zedd0202** [rs/SDWebImage#2239]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [o] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. **ios** / macos / tvos / watchos
 Platform Version        | e.g. **11.0** / 10.13.0 / 11.0 / 4.0
 SDWebImage Version      | e.g. 4.2.0 / 4.1.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 9 / Xcode 8
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

First, please understand that grammar is not natural.I do not get any warnings or errors, but I'm just wonder.
`SDWebImageDownloader.shared().downloadImage(with: URL(string: url), options: SDWebImageDownloaderOptions.lowPriority,......so on`
I set the option to lowpriority, Does this mean do not get the cache stored in memory or disk?
According to the document, SDWebImageDownloaderUseNSURLCache is set by default, but if I give the lowpriority option, Could not load the cached image? Do I only need to give the SDWebImageDownloaderUseNSURLCache option to load the cached image?
or getting the cached image **automatically** without the SDWebImageDownloaderUseNSURLCache option?

**Comment by dreampiggy** [comment]
`SDWebImageDownloaderOptions.lowPriority` just means to use `NSURLSessionTaskPriorityLow` in `NSURLSessionTask`. No other effect will take.

This is not related to the cache options because it's just a hint for URLSession in downloading.  `SDWebImageDownloaderUseNSURLCache` means to use `NSURLRequestUseProtocolCachePolicy` instead of `NSURLRequestReloadIgnoringLocalCacheData`, just download from the original source.

If you want to use `SDImageCache`, you should not use `SDWebImageDownloader`, you should use `SDWebImageManager`, which is a bridge between cache and downloader.
